doc/rst/language/spec/procedures.rst

```diff
@@ -1544,11 +1544,10 @@ the following rules in order:
    formal with the same width is better. Each of the following bullets
    represents a different numeric width for this rule:
 
-   * All numeric types that match the default width as well as all
-     ``bool`` types. This includes ``bool``, ``bool(?w)``, ``int``
-     ``uint`` ``real`` ``imag`` ``complex`` as well as their more
-     specific names ``int(64)`` ``uint(64)`` ``real(64)`` ``imag(64)``
-     ``complex(128)``
+   * All numeric types that match the default width, as well as
+     ``bool``.  This includes ``int`` ``uint`` ``real`` ``imag``
+     ``complex`` as well as their more specific names ``int(64)``
+     ``uint(64)`` ``real(64)`` ``imag(64)`` ``complex(128)``
```

Comment:
Intuitively, I'd like to keep "\`\`bool\`\`" after "This includes." However OK to keep as-is.